docs/src/getting-started/uninstallation.md

```diff
@@ -0,0 +1,65 @@
+# Uninstallation
+## Linux & macOS
+Use the `asdf` docs to learn how to remove the Starknet Foundry plugin [here](https://asdf-vm.com/manage/plugins.html#remove)
+
+To ensure Starknet Foundry is fully removed, run:
+
+```bash
+snforge --version
+```
+
+If the command is not found, the uninstallation was successful.
+
+## Windows
+### Remove Starknet Foundry Files
+You can delete Starknet Foundry manually by navigating to:
+
+```
+📂 C:\Users\YourUsername\AppData\Local\Programs\snfoundry
+```
+
+Right-click the folder and select Delete.
+
```

Comment:
This path is technically not always valid, it depends on where exactly user installed it. I'd just say something along the lines of "Delete the directory in which you extracted the binary". Maybe we can either say that it is usually `%LOCALAPPDATA%\Programs\snfoundry` or link to the relevant installation section so user can cross reference it.